podcasts/End of Year/Stories/IntroStory.swift

```diff
@@ -7,20 +7,38 @@ struct IntroStory: StoryView {
     var body: some View {
         GeometryReader { geometry in
             ZStack {
-                VStack(spacing: 0) {
-                    Image("2022_big")
-                        .resizable()
-                        .aspectRatio(contentMode: .fit)
-                        .padding(.top, geometry.size.height * Constants.imageVerticalPadding)
+                ZStack {
+                    TwentyThree()
 
-                    let title = L10n.eoyStoryIntroTitle.replacingOccurrences(of: "...", with: "!")
-                    StoryLabel(title, for: .title)
-                        .padding(.top, Constants.spaceBetweenImageAndText)
+                    Image("2023-title")
```

Comment:
I have an idea, if we apply the [`IconParallaxModifier`](https://github.com/Automattic/pocket-casts-ios/blob/bffe307dea08c50d6188ad9c646c0fca06dddc31/podcasts/Onboarding/Patron/PatronAppIconUnlock.swift#L440) we get a subtle parallax effect when the device is rotated that really adds to the depth effect. 


https://github.com/Automattic/pocket-casts-ios/assets/793774/34255baa-ed14-4d84-be2b-8968c1fe1ca1

Reply:
Nice! Done in 555a931679492002c67ea286f9e454f0f0710275